wiki/osu!supporter/tr.md

```diff
@@ -0,0 +1,60 @@
+---
+outdated: true
+---
+
+<!-- TODO: website updates, needs a review on the writing style -->
+
+# osu!supporter
+
+*Siteden osu!supporter sayfasına gitmek için, bakınız: [oyunu destekle](https://osu.ppy.sh/home/support)*
+
+## Ayrıcalıklar
+
+*osu!supporter olarak aldığınız ayrıcalıkların listesi için, bakınız: [oyunu destekle](https://osu.ppy.sh/home/support)*
+
+Kısacası, osu!supporter etiketi satın alarak, bazı extra ayrıcalıkların yanı sıra oyunun geliştirilmesini doğrudan desteklemiş olacaksınız. osu!supporter sayesinde ettiğiniz extra ayrıcalıkların oynanışı, performansını, ya da skoru herhangi bir şekilde etkilemeyeceğini aklınızda tutmalısınız.
+
+## osu!supporter elde etmek
+
+![osu!store ürün afişi](img/store-product.jpg "osu!store'daki, osu!supporter ürün afişi")
+
+osu!supporter elde etmek için, giriş yapın (ya da [kayıt olun](/wiki/sign_up)), ve ardından [mağaza sayfası](https://osu.ppy.sh/store/products/supporter-tag) gidin. Kaydırıcıyı ya da aylık tuşları (kaydırıcının altında) kullanarak osu!supporter etiketinizin süresini seçiniz. Farklı bir süre ödeyeceğiniz miktarı değiştirecektir. Kullanılan para birimi Amerika Birleşik Devletleri dolarıdır (USD). Eğer ülkeniz farklı bir para birimi kullanıyor ise, lütfen, [OANDA](https://www.oanda.com/currency/converter/) ya da [Google](https://www.google.com/search?q=usd+exchange+rate) gibi para birimi dönüştürücülerini kullanarak ödeyeceğiniz fiyatı kontrol edin. osu!store sayfasında sunulan ücretler her bir hesap için ayrıdır.
+
+### Süreyi seçmek
+
+![osu!supporter satın almak](img/selecting-duration.jpg "osu!supporter için kullanıcıyı ve süreyi seçmek")
+
+Kaydırıcının üzerinde bulunan kullanıcı kartı osu!supporter aldığınız kullanıcıyı gösterir. Varsayılan olarak, bu sizin kullanıcı kartınız olmalı. Bir arkadaşınıza hediye etmek için, arkadaşınızın kullanıcı ismini (Kullanıcı ID'sini değil) kullanıcı kartının altındaki gri kutuya girin. Bu yapıldıktan sonra, kullanıcı kartı girilen kullanıcı isminin hesabıyla değiştirilecektir. Buradan sonra, hediye etmek istediğiniz osu!supporter süresini seçebilirsiniz. Ancak, birden fazla kişiye gönderilen hediyelerde bir promosyon yapılmamaktadır.
+
+Bu ekrandan tek seferde sadece bir kişiye osu!supporter hediye edebilirsiniz. Başka bir kullanıcıya hediye etmek için, [osu!supporter mağaza sayfasına](https://osu.ppy.sh/store/products/supporter-tag) geri gidip başka bir kullanıcıyı eklemek için gereken adımları tekrarlamalısınız.
+
+### Ödeme yapmak
+
+İşiniz bittiği zaman, `Sepete Ekle` tuşuna basarak osu!store mağaza sepetinize ekleyin.
+
+![osu!store mağaza sepeti](img/shopping-cart.jpg "içinde flyte için osu!supporter bulunan bir osu!store mağaza sepeti")
+
+Satın alımı yapmak için hazır olduğunuz zaman, [mağaza sepetinize](https://osu.ppy.sh/store/cart) ve `Ödeme` tuşuna basın. Ardından adımları takip ederek ödemenizi gerçekleştirin.
```

Comment:
```suggestion
Satın alımı yapmak için hazır olduğunuz zaman, [mağaza sepetinize](https://osu.ppy.sh/store/cart) gidin ve `Ödeme` tuşuna basın. Ardından adımları takip ederek ödemenizi gerçekleştirin.
```
seems like this was deleted by accident while I was adding my suggestions, my bad